### DISCUSSION TITLE
Did not find build matching uploaded version yet

### DISCUSSION THREAD

**Discussion by hzfplay** [codemagic-ci-cd/codemagic-docs#2490] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

Error prompt:
Did not find build matching uploaded version yet, it might be still processing. Waiting to try again...
The application binary was successfully uploaded to App Store Connect for processing but the corresponding build is not available in App Store Connect. This usually happens when the uploaded application has one or more issues. Please check your email for direct feedback from Apple.
I have searched for relevant forums and documents, and they all say that it is a problem with the version code. May I ask how to set the version number

### Expected results

I tried setting many different version code, but they still didn't work

### Actual results

Waiting for 15 minutes for the upload process to finally end

### Build id (optional)

66274f49828f4761964415b9

**Comment by himesh-cm** [comment]
Based on the information you provided, have you received any emails from Apple? If so, please share the details so we can investigate further.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
